android/assets/jsons/translations/template.properties

```diff
@@ -184,6 +185,7 @@ Provides [amountOfCulture] culture at 30 Influence =
 Provides 3 food in capital and 1 food in other cities at 30 Influence = 
 Provides 3 happiness at 30 Influence = 
 Provides land units every 20 turns at 30 Influence = 
+Provides [amount] [stat] per turn = 
```

Comment:
I believe this should be added automatically when generating the translation files, right?